spec.html

```diff
@@ -5853,10 +5853,11 @@
             [[TemplateMap]]
           </td>
           <td>
-            A List of Record { [[Strings]]: List, [[Array]]: Object}.
+            A List of Record { [[Site]]: Parse Node, [[Array]]: Object}.
           </td>
           <td>
-            Template objects are canonicalized separately for each realm using its Realm Record's [[TemplateMap]]. Each [[Strings]] value is a List containing, in source text order, the raw String values of a |TemplateLiteral| that has been evaluated. The associated [[Array]] value is the corresponding template object that is passed to a tag function.
+            Template objects are canonicalized separately for each realm using its Realm Record's [[TemplateMap]]. Each [[Site]] value is a Parse Node containing a |TemplateLiteral|. The associated [[Array]] value is the corresponding template object that is passed to a tag function.
```

Comment:
Rather than "a Parse Node containing a |TemplateLiteral|", it would be more consistent (with usage in the rest of the spec) to say something like:
- "a Parse Node that is a |TemplateLiteral|", or
- "a Parse Node that is an instance of |TemplateLiteral|", or
- "a |TemplateLiteral| (i.e., a Parse Node)".

Reply:
Done